config/seata-config-core/src/main/java/io/seata/config/FileConfiguration.java

```diff
@@ -197,6 +183,20 @@ public void run() {
                     return;
                 }
                 try {
+                    long tempLastModified = new File(targetFilePath).lastModified();
+                    if (tempLastModified > targetFileLastModified) {
+                        Config tempConfig;
+                        if (name.startsWith(SYS_FILE_RESOURCE_PREFIX)) {
+                            Config appConfig = ConfigFactory.parseFileAnySyntax(new File(targetFilePath));
+                            tempConfig = ConfigFactory.load(appConfig);
+                        } else {
+                            tempConfig = ConfigFactory.load(name);
+                        }
+                        if (null != tempConfig) {
+                            fileConfig = tempConfig;
+                            targetFileLastModified = tempLastModified;
+                        }
+                    }
```

Comment:
FileConfiguration  test well.
But we use FileConfiguration to load registry.conf and file.conf, so both listen to their changes, Is it necessary to monitor them all?

Reply:
registry.conf is that the metadata configuration is not allowed to change at run time. If it is allowed to change, for example, if I use the Nacos registry, I need to reload the entire instance.

Reply:
fixed.